javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -25,6 +25,101 @@ public void shouldNotBeInstantiable() {
         AssertionsExtensions.assertThat(API.class).isNotInstantiable();
     }
 
+    //
+    // Alias should return not null.
+    // More specific test for each aliased class implemented in separate test class
+    //
+
+    @Test
+    public void shouldFunction0ReturnNotNull() {
+        assertThat(Function0(() -> null)).isNotNull();
```

Comment:
For me this is ok. When this PR is pulled I will get my hands dirty and add more tests.

Example:

```java
assertThat(Valid(1)).isEqualTo(Validation.valid(1));